Cargo.toml

```diff
@@ -66,3 +66,6 @@ predicates = "3.0"
 pretty_assertions = "1.4"
 ring = "0.16.20"
 hex = "0.4.3"
+
+[workspace]
+members = [".", "registry"]
```

Comment:
does this affect the publishing of `buffrs`?

Reply:
It does not — when you do `cargo publish`, it always only publishes the crate you are currently in. You can verify this by doing a `cargo publish --dry-run`.

When we add a dependency from `registry` to `buffrs`, we can do it like this:

```toml
buffrs = { path = "..", version = "0.1.0" }
```

What this does is that if you are building locally, it'll just use whatever the path points to (it validates that it is the right version). But if you are publishing the registry (should we want to do that), then it strips out the `path` and goes only by the `version`. That way it's easy to work locally but when we publish it does what it should.